metrics/config_test.go

```diff
@@ -835,7 +507,6 @@ func TestUpdateExporter(t *testing.T) {
 func TestUpdateExporterFromConfigMapWithOpts(t *testing.T) {
 	setCurMetricsConfig(nil)
 	oldConfig := getCurMetricsConfig()
-	successTestsInit()
```

Comment:
Do we still need `successTestsInit`? I've seen it removed several places here.

Reply:
It's removed - in the diff